db/migrations/0009_add_wallet_details_table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS wallet_details (
+  wallet_address TEXT PRIMARY KEY,
+  is_sanctioned BOOLEAN NOT NULL DEFAULT FALSE
+);
```

Comment:
I am wondering if we should use the same data schema as was proposed in https://github.com/filcdn/worker/pull/143/files#diff-9ae133790f3d9c9aabc53407feba0ca412ba43527ceb8cf3cb5a1896ddcc73dc?


```sql
CREATE TABLE IF NOT EXISTS wallet_details (
  wallet_address TEXT PRIMARY KEY,
  sanction_check_status TEXT NOT NULL DEFAULT 'pending'
    CHECK (sanction_check_status IN ('sanctioned', 'approved', 'pending'))
);
```

Reply:
Maybe we can set `is_sanctioned = NULL` to indicate that we haven't checked the sanction status yet?

I think we can also drop the `wallet_` prefix from the column name, since the table name `wallet_details` already indicates that the address relates to wallets.

Just ideas to consider, please decide for yourself which of these suggestions make sense and which to ignore (or defer for later).

Reply:
Changed column name to `address` in https://github.com/filcdn/worker/pull/171/commits/4ec2fe56c413aea782b57e0be3407ec55bdc6eb3